src/db/addStudentMgmt.sql

```diff
@@ -349,10 +361,13 @@ CREATE OR REPLACE FUNCTION getStudentIDByIssuedID(schoolIssuedID VARCHAR(50))
 RETURNS INT
 AS
 $$
-BEGIN
-   RAISE WARNING 'Function not implemented';
-END
-$$ LANGUAGE plpgsql
+
+SELECT s.id
+FROM student s
+WHERE TRIM(s.schoolissuediD) ILIKE TRIM($1);
+
+$$ LANGUAGE sql
+
```

Comment:
Extra new line on 370? (Might also choose to fix it during the review tomorrow.)

Reply:
Extra lines removed here and a couple other places.  In either case, a good think to check over, overall, during the review tomorrow.